conda/plugins/manager.py

```diff
@@ -361,6 +378,33 @@ def invoke_post_solves(
         for hook in self.get_hook_results("post_solves"):
             hook.action(repodata_fn, unlink_precs, link_precs)
 
+    def load_configuration_parameters(self) -> None:
+        """
+        Iterates through all registered configuration parameters and adds them to the context class.
+
+        :raises PluginError: Happens when plugins attempt to override existing parameters
+
+        TODO:
+            - Need to add configuration parameters to ``Context.category_map`` and
+              ``Context.description_map``
```

Comment:
Could these because dynamic properties?

Reply:
I think they already are by using the `@property` decorator.

We just need to implement a proper setters for these properties, so we can update them.